staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -62,10 +62,16 @@ var baseOpts = []VersionedOptions{
 			library.URLs(),
 			library.Regex(),
 			library.Lists(),
+
```

Comment:
The PR description says "CostEstimatorOptions should be added at least one release earlier". I am expecting it to be in 1.28 instead of 1.0, so here is a bit surprise. Could you comfirm?

Reply:
Ah. I see this discussion in a previous thread. Could you mention it in the comments

Reply:
I have added a comment to further explain here.